testing/google-cloud-appengine-flex-java/src/main/appengine/app.yaml

```diff
@@ -0,0 +1,19 @@
+runtime: java
+env: flex
+
+runtime_config:  # Optional
```

Comment:
No we don't, we actually don't even need that section (since it is optional =). Removed it.